### PR TITLE
handle layout of shorts thumbnails

### PR DESCRIPTION
**Pull request by kskarthik**
# Before 
image

# After
image

### CODE REVIEW

**Comment by kskarthik** [comment]
Should close #1126

**Comment by kskarthik** [comment]
Some variations of the shorts thumbnail

image

**Comment by kskarthik** [comment]
@FireMasterK what are your thoughts?

**Comment by ju-sh** [comment]
Am a random user. Just my 2 cents.

It certainly became better. But the thumbnails did become distorted (I guess it's aspect ratio?).

Would it be possible to scale it without the distortion?

**Comment by kskarthik** [comment]
@ju-sh  shorts thumbnails come in various sizes and shapes, just check the screenshort below, if i try to fix some short thumbnails, some other break

image

File: src/components/VideoItem.vue [hunk 1]
```diff
@@ -10,13 +10,25 @@
...
+                :alt="video.title"
+                loading="lazy"
+            />
             <div class="relative text-sm">
                 <span
                     class="thumbnail-overlay thumbnail-right"
                     v-if="video.duration > 0"
                     v-text="timeFormat(video.duration)"
                 />
+                <span class="thumbnail-overlay thumbnail-right" v-if="video.duration <= 60" v-text="`SHORTS`" />
```
Comment (FireMasterK) [inline src/components/VideoItem.vue:26]: This should ideally be translatable.

Comment (kskarthik) [reply src/components/VideoItem.vue]: @FireMasterK done, please check https://github.com/TeamPiped/Piped/pull/1141/commits/00aa83b802cf19c469a64f1792a70177d28fb64b

Comment (kskarthik) [reply src/components/VideoItem.vue]: ok, found it, check https://github.com/TeamPiped/Piped/pull/1141/commits/18385ec7759988268dce7eef891ab5a507f7d492

Comment (kskarthik) [reply src/components/VideoItem.vue]: ## Final design 

image

Comment (kskarthik) [reply src/components/VideoItem.vue]: Hey, can we use the shorts logo instead of text? https://en.wikipedia.org/wiki/File:Youtube_shorts_icon.svg

Comment (kskarthik) [reply src/components/VideoItem.vue]: @FireMasterK I hope we're good to go

**Comment by FireMasterK** [comment]
Whoops!

**Pull request merged**